model/telemetry/registry.yaml

```diff
@@ -81,3 +81,42 @@ groups:
         brief: >
           The version string of the auto instrumentation agent or distribution, if used.
         examples: ["1.2.3"]
+      - id: telemetry.sdk.processor.type
```

Comment:
Wondering whether that should be an enum instead. That would allow to specify well-known values like proposed with `batch-span` but also allow for custom values.

Reply:
Fixed in 34e56e7